src/power_grid_model_io/converters/base_converter.py

```diff
@@ -20,10 +25,17 @@
 class BaseConverter(Generic[T], ABC):
     """Abstract converter class"""
 
-    def __init__(self, source: Optional[BaseDataStore[T]] = None, destination: Optional[BaseDataStore[T]] = None):
+    def __init__(
+        self,
+        source: Optional[BaseDataStore[T]] = None,
+        destination: Optional[BaseDataStore[T]] = None,
+        log_level: int = logging.ERROR,
+    ):
         """
         Initialize a logger
         """
+        logging.basicConfig(level=log_level)
+        structlog.configure(wrapper_class=structlog.make_filtering_bound_logger(log_level))
```

Comment:
i don't know whether this is the case, but does this only override the local config within this class or the global config? if the latter, could it also affect users' config?

Reply:
Just tried, not the global config rather than local instances: two logger script running at the same time could output their respective log level correctly

Reply:
Conversation resolved.